frontend/packages/client/src/App.js

```diff
@@ -18,15 +30,18 @@ function App() {
       )}
       onReset={() => (window.location.href = '/')}
     >
-      <Web3Provider network={process.env.REACT_APP_FLOW_ENV}>
-        <NotificationModalProvider>
-          <Router>
-            <ErrorHandler>
-              <AppPages />
-            </ErrorHandler>
-          </Router>
-        </NotificationModalProvider>
-      </Web3Provider>
+      <QueryClientProvider client={queryClient}>
+        <Web3Provider network={process.env.REACT_APP_FLOW_ENV}>
+          <NotificationModalProvider>
+            <Router>
+              <ErrorHandler>
+                <AppPages />
+              </ErrorHandler>
+            </Router>
+          </NotificationModalProvider>
+        </Web3Provider>
+        {isLocalDevMode && <ReactQueryDevtools />}
```

Comment:
This enables the react-query debugger: 
<img width="809" alt="Screen Shot 2022-07-07 at 17 34 57" src="https://user-images.githubusercontent.com/7526740/177867813-fbafbfd2-5d0e-4e32-ae5b-4cf707df6225.png">